app/assets/stylesheets/contacts.scss

```diff
@@ -1,5 +1,7 @@
 .page-contacts {
-  .sidebar { padding-bottom: 10px; }
+  .sidebar { 
```

Comment:
Line contains trailing whitespace

Reply:
First: Thanks for the fixes. :+1: 

You don't change anything here anymore, so you can just revert this to how it was before (which would also fix this comment from @diaspora-code-review). And when you are at it, you can also remove the other changes which aren't related to the fix (you added and remove some newlines at some places). And when you are finished with the cleanup, please squash your commits to a single commit.